### ISSUE TITLE
How to properly utilize external flake outputs?

### ISSUE THREAD

**Issue by rpgwaiter** [divnix/devos#341]
I use flakes to manage all of my code projects. They output packages. How can I use them in my devos-powered config? I added my projects to the inputs of pkgs/flake.nix, but the documentation doesn't seem to mention how to handle flake inputs.

**Comment by cipharius** [comment]
I haven't used this feature myself yet, but looking at documentation the provided example does show how to access the declared sources.

The example `libinih` package asks for `srcs`, from which it then inherits the `libinih` from declared inputs.

**Comment by blaggacao** [comment]
Unfortunately, the documentation is really a bit outdated (w.r.t. `develop`), since we started out-factoring `digga`. And `core` has fallen really behind. Apologies for the inconvenience.

Digga has gotten a bit of an initial version of a documentation at https://digga.divnix.com, but in devos, we still need to merge `develop` back into `core` and completely revise the docs.

In `develop` we implemented nvfetcher for the source fetching, since it knows how to track a variety of remote sources for updates.
